### PR TITLE
Proposed fix for issue #40: add option to include files w/o extensions

### PR DESCRIPTION
**Pull request by glenpike**
Create new command line option called --include-extensionless
files that will override the path_filter where a path without
an extension is encountered.

### CODE REVIEW

File: src/codemod.py [hunk 1]
```diff
@@ -80,6 +83,15 @@
 
 
 import sys, os
+def is_extensionless(path):
+  """
+  Returns True if path has no extension.
+  """
```
Comment (modocache) [inline src/codemod.py]: It'd be great to have some doctests here, since those are the only kinds of tests in codemod at the moment.

**Comment by modocache** [comment]
This looks great--thanks, @glenpike! :+1:

I'd love to have a few more doctests before merging this in, though.

**Comment by glenpike** [comment]
Okay, I'll have a look at sorting those out.

**Comment by modocache** [comment]
Thanks again, @glenpike! :+1:

**Pull request merged**